shell/client/_topbar.scss

```diff
@@ -459,9 +436,43 @@ body>.topbar {
 
   >.navbar {
     @media #{$desktop} {
-      &.hide-desktop {
+      button.navbar-shrink {
+        @extend %unstyled-button;
+        display: block;
+        width: 100%;
+        height: $navbar-shrink-button-height;
+        border: 0px solid #303030;
+        border-left-width: ($navbar-width-desktop - $navbar-width-desktop-shrunk);
+        background-color: #606060;
+        color: #ddd;
+        text-align: center;
+        line-height: $navbar-shrink-button-height;
+        padding: 0;
+        outline: none;
```

Comment:
If you remove this `outline: none`, you'll have a focus ring like everything else interactive.